jwt/src/main/java/me/uport/sdk/jwt/JWTTools.kt

```diff
@@ -212,6 +207,29 @@ class JWTTools(
             throw InvalidJWTException("JWT has expired: exp: ${payload.exp}")
         }
 
+        if (payload.aud != null) {
+
+            if (UniversalDID.canResolve(payload.aud)) {
+
+                if (aud == null) {
+                    throw InvalidJWTException("JWT audience is required but your app address has not been configured")
+                }
+
+                if (!aud.equals(payload.aud)) {
```

Comment:
please use `!=` instead of `!equals`
on kotlin it compares contents, not references
```suggestion
                if (aud != payload.aud) {
```